src/general.jl

```diff
@@ -41,8 +41,34 @@ function clean_abundance_tables(files::Array{String, 1};
 end
 
 
+"""
+    rm_strat!(df::DataFrame; col::Union{Int, Symbol}=1)
+
+Given an abundance table of functional profiles ex. humann2 file, removes stratification to show total abundances.
+```jldoctest rm_strat!
+Examples
+≡≡≡≡≡≡≡≡≡≡
+julia> table = CSV.read("test/files/humanntestfile.tsv", DataFrame)
```

Comment:
This is great, but it doesn't look like this file got added / committed

Reply:
I just added it! I though it got added when I did commit -am ... but I am assuming new files need to get added explicitly?